tests/cli/test_cmds_spark_run.py

```diff
@@ -284,6 +288,40 @@ def test_get_spark_env(
     )
 
 
+def test_get_spark_env_load_iam_creds():
+    data = """
+[default]
+aws_access_key_id = <foo-id>
+aws_secret_access_key = <foo-secret>
+    """
+    with mock.patch(
+        "paasta_tools.cli.cmds.spark_run.open",
+        return_value=io.StringIO(data),
+        autospec=None,
+    ) as iam_creds_file:
```

Comment:
i think we generally name mocks `mock_*` - so for consistency this is probably better off as `mock_open` or something similar